bindings/nodejs/examples/how_tos/native_tokens/mint.ts

```diff
@@ -25,40 +21,27 @@ async function run() {
         // Get the account we generated with `01-create-wallet`
         const account = await wallet.getAccount('Alice');
 
-        const balance = await account.sync();
-
-        // We can first check if we already have an alias in our account, because an alias can have many foundry outputs and therefore we can reuse an existing one
-        if (balance.aliases.length > 0) {
-            // If we don't have an alias, we need to create one
-            const transaction = await account
-                .prepareCreateAliasOutput()
-                .then((prepared) => prepared.send());
-            console.log(`Transaction sent: ${transaction.transactionId}`);
+        // May want to ensure the account is synced before sending a transaction.
+        let balance = await account.sync();
 
-            // Wait for transaction to get included
-            const blockId = await account.retryTransactionUntilIncluded(
-                transaction.transactionId,
-            );
+        // Find first foundry and corresponding token id
+        const tokenId = balance.foundries[0];
 
```

Comment:
no foundries check? Or as the example explains an "existing token with its foundry" we assume its ok?

Reply:
Added the check.